UpTraceCloudFunctions/acs/index.js

```diff
@@ -3,6 +3,9 @@ const {SecretManagerServiceClient} = require('@google-cloud/secret-manager');
 const jwt = require('jsonwebtoken');
 const validator = require('@authenio/samlify-node-xmllint');
 const saml = require('samlify');
+
+const controller = require('./Controllers/HelloController')
+
```

Comment:
Ideally you would use a separate `api-gateway.js` file that creates a "router" object that receives the request and only requires the controller after determining which controller the request is for